grammars/python.cson

```diff
@@ -1151,7 +1151,7 @@
         'name': 'string.quoted.double.block.sql.python'
         'patterns': [
           {
-             'begin': '(?=\\s*(SELECT|INSERT|UPDATE|DELETE|CREATE|REPLACE|ALTER))'
+             'begin': '(?=\\s*(SELECT|INSERT|UPDATE|DELETE|CREATE|REPLACE|ALTER|WITH))'
```

Comment:
NM, I found it. Looks like the order of some things is a little wonky in the fourth case.